manage.py

```diff
@@ -6,7 +6,7 @@
     if 'test' in sys.argv:
         os.environ['DJANGO_SETTINGS_MODULE'] = 'icommons_ext_tools.settings.test_settings'
     else:
-        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.local')
+        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.aws')
```

Comment:
OK, I'll add it to the vagrant setup then as part of this PR and let you know when it's ready to go.